app/js/states/maintenance/maintenance.controller.js

```diff
@@ -154,6 +155,8 @@ function MaintenanceCtrl(
     };
 
     $scope.category = PreferenceService.get(CATEGORY_PREFERENCE_KEY) || DEFAULT_CATEGORY;
+    $scope.category = $location.search().maintenanceCategory ?
+                      $location.search().maintenanceCategory : $scope.category;
```

Comment:
you can consolidate this by 

$scope.category = $location.search().maintenanceCategory || PreferenceService.get(CATEGORY_....

Reply:
Also, having $scope.category in MaintenanceCtrl and a separate reference in maintenanceCategorySelect that stays in sync with this reference is weird to me. Shouldn't we just use this reference in maintenanceCategorySelect?

Reply:
@Kinlaw you are right about $scope.category.